test/python/scheduler/test_basic_scheduler.py

```diff
@@ -280,3 +288,16 @@ def test_barriers_in_middle(self):
             self.assertEqual(actual[0], expected[0])
             self.assertEqual(actual[1].command, expected[1].command)
             self.assertEqual(actual[1].channels, expected[1].channels)
+
```

Comment:
try some tests built like the others, which include:
Measure qubits 0, 1, 2 into different mem slots (use FakeOpenPulse3Q backend)
Measure a subset of 0, 1, 2 into different memory slots (try measuring just 1, and in another test just 2, out of the three)
You can also try multiple measurements on the same qubit with a gate in between.
Be creative :)